bokehjs/src/coffee/models/widgets/main.coffee

```diff
@@ -1,34 +1,36 @@
 module.exports = {
-  editors:                  [require('./cell_editors'), 'Editor']
-  formatters:               [require('./cell_formatters'), 'Formatter']
+  models: {
```

Comment:
OK, good time to make this consistent, but let's add a `MIGRATION` tag and note it